runtime/seccomp_filter.c

```diff
@@ -47,22 +48,71 @@ struct sock_filter ia2_filter[] = {
     BPF_SYSCALL_POLICY(exit_group, ALLOW),
     BPF_SYSCALL_POLICY(fcntl, ALLOW),
     BPF_SYSCALL_POLICY(futex, ALLOW),
+    BPF_SYSCALL_POLICY(sysinfo, ALLOW), // simple1 uses sysinfo inside qsort inside CRYPTO_THREAD_run_once
+    BPF_SYSCALL_POLICY(getcwd, ALLOW),
+    BPF_SYSCALL_POLICY(lseek, ALLOW),
+    BPF_SYSCALL_POLICY(syslog, ALLOW),
+    BPF_SYSCALL_POLICY(getdents, ALLOW),
+    BPF_SYSCALL_POLICY(getdents64, ALLOW),
+    BPF_SYSCALL_POLICY(getegid, ALLOW),
+    BPF_SYSCALL_POLICY(geteuid, ALLOW),
+    BPF_SYSCALL_POLICY(getgid, ALLOW),
+    BPF_SYSCALL_POLICY(getuid, ALLOW),
+    BPF_SYSCALL_POLICY(getppid, ALLOW),
+    BPF_SYSCALL_POLICY(setpgid, ALLOW), // used by criterion
+    BPF_SYSCALL_POLICY(gettid, ALLOW),
     BPF_SYSCALL_POLICY(getpid, ALLOW),
     BPF_SYSCALL_POLICY(getrandom, ALLOW),
+    BPF_SYSCALL_POLICY(prctl, ALLOW),
+    BPF_SYSCALL_POLICY(ioctl, ALLOW),
```

Comment:
`ioctl` and friends deserves a comment at least, we can't just allow all ioctls through, can we? Or are we sandboxing what devices can be opened? Not saying we should do this now, just don't want to forget.

Reply:
Good point; this is needed by Criterion but we can fairly easily limit it to only the one `ioctl` that Criterion actually performs.

Reply:
Done in b449ff0.